sources/packages/web/src/components/institutions/request-a-change/CompletedOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+              </template>
+            </v-data-table-server>
+          </toggle-content>
+        </content-group>
+      </template>
+    </body-header-container>
+  </tab-container>
+</template>
+
+<script lang="ts">
+import { ref, watch, defineComponent } from "vue";
+import {
+  DEFAULT_PAGE_LIMIT,
+  DataTableOptions,
+  PaginatedResults,
+  CompletedOfferingChangeSummaryHeaders,
+  DataTableSortByOrder,
+  DEFAULT_DATATABLE_PAGE_NUMBER,
+} from "@/types";
+import { useFormatters } from "@/composables";
+import { CompletedApplicationOfferingChangesAPIOutDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestService } from "@/services/ApplicationOfferingChangeRequestService";
+import StatusChipApplicationOfferingChange from "@/components/generic/StatusChipApplicationOfferingChange.vue";
+
+export default defineComponent({
+  components: { StatusChipApplicationOfferingChange },
+  props: {
+    locationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const loading = ref(false);
+    const searchCriteria = ref("");
+    const { dateOnlyLongString } = useFormatters();
+    const applications = ref(
+      {} as
+        | PaginatedResults<CompletedApplicationOfferingChangesAPIOutDTO>
+        | undefined,
+    );
+    let currentPage = NaN;
+    let currentPageLimit = NaN;
+
+    /**
+     * Load completed applications offering change records for institution.
+     * @param page page number, if nothing passed then {@link DEFAULT_DATATABLE_PAGE_NUMBER}.
+     * @param pageCount page limit, if nothing passed then {@link DEFAULT_PAGE_LIMIT}.
+     * @param sortField sort field, if nothing passed then api sorts with application number.
+     * @param sortOrder sort oder, if nothing passed then {@link DataTableSortByOrder.ASC}.
+     */
+    const getCompletedSummaryList = async (
+      page = DEFAULT_DATATABLE_PAGE_NUMBER,
+      pageCount = DEFAULT_PAGE_LIMIT,
+      sortField?: string,
+      sortOrder?: DataTableSortByOrder,
+    ) => {
+      loading.value = true;
+      applications.value =
+        await ApplicationOfferingChangeRequestService.shared.getCompletedApplications(
+          props.locationId,
+          {
+            page,
+            sortField,
+            sortOrder,
+            pageLimit: pageCount,
+            searchCriteria: searchCriteria.value,
+          },
+        );
+      loading.value = false;
+    };
+
+    // Pagination sort event callback.
+    const paginationAndSortEvent = async (event: DataTableOptions) => {
+      currentPage = event.page;
+      currentPageLimit = event.itemsPerPage;
+      const [sortByOptions] = event.sortBy;
+      await getCompletedSummaryList(
+        event.page,
+        event.itemsPerPage,
+        sortByOptions?.key,
+        sortByOptions?.order,
+      );
+    };
+
+    // Search table.
+    const searchApplicationOfferingChangeRecords = async () => {
+      // Fix for the search pagination issue.
```

Comment:
instead of commenting that this is a fix, wouldn't be better comment why it fixes / what is the issue?

Reply:
I added it in the my previous PR description , i will add it to one of the comment in the code too